packages/@okta/vuepress-site/docs/guides/implement-saml2/setup-app/index.md

```diff
@@ -8,7 +8,7 @@ You can use either an existing OpenID Connect app integration or create a new on
 
 1. In the Admin Console, go to **Applications** > **Applications**.
 1. Click **Create App Integration**.
-1. On the Create a new app integration window, select **SAML 2.0** as the **Sign-in method**. Select **Native Application** as the **Application type**. Click **Next**.
+1. On the Create a new app integration window, select **OIDC - OpenID Connect ** as the **Sign-on method**. Select **Native Application** as the **Application type**. Click **Next**.
```

Comment:
Correct to:
On the Create a new app integration window, select **OIDC - OpenID Connect ** as the **Sign-in method**. Select **Native Application** as the **Application type**. Click **Next**.